content/millennium/r4/documents/binary.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Binary | R4 API
+---
+
+# Binary
+
+* TOC
+{:toc}
+
+## Overview
+
+The Binary resource can contain any clinical content such as text, images, and PDFs.  This resource is currently limited to clinical documents and diagnostic reports.  The diagnostic reports are currently limited to Anatomic Pathology and Radiology.
+
+It is recommended to request all Binary resources only after obtaining a link to the resource via references from DiagnosticReport or DocumentReference. It is not recommended to start a workflow in the Binary resource. 
+
+The consumer must populate the Accept header with either application/fhir+json or the format returned in the attachment.contentType of the referring resource.  If the Accept header is application/fhir+json, a FHIR Binary resource is returned with the raw data populated in the content. Otherwise, the raw data will be returned (not contained within a FHIR resource). For more information, see [the Binary Documentation](http://hl7.org/fhir/r4/binary.html#rest).
```

Comment:
Update "content" to "data field"  The name of the field changed in R4.

https://www.hl7.org/fhir/r4/binary-definitions.html#Binary.data

Reply:
https://github.com/cerner/fhir.cerner.com/commit/4e4c61099afb22bb16dca0268cef934957bc4bc6

Reply:
you changed this to 'data', not 'data field' as requested

Reply:
This was updated as below so there was no need data field I believe
```
The consumer must populate the Accept header with either `application/fhir+json` or the format returned in the `attachment.contentType` of the referring resource. If the Accept header is `application/fhir+json`, a FHIR Binary resource is returned with the raw data populated in `Binary.data`. If the Accept header is anything else, the raw data will be returned instead of a FHIR Binary resource. For more information, see the HL7<sup>®</sup> FHIR<sup>®</sup> [Binary documentation](http://hl7.org/fhir/r4/binary.html#rest).
```